{"name": "test_auth_not_existing_courier", "status": "passed", "steps": [{"name": "Логин с неверным логином и паролем.", "status": "passed", "start": 1724619067128, "stop": 1724619067411}], "start": 1724619067128, "stop": 1724619067411, "uuid": "7c23d5a4-a178-4cd3-a751-dd05f594bbc9", "historyId": "d7c333e5123d80a524f78768202613bf", "testCaseId": "d7c333e5123d80a524f78768202613bf", "fullName": "tests.test_login_user.TestAuthUser#test_auth_not_existing_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "LAPTOP-4UNN3S2K"}, {"name": "thread", "value": "2996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}